PlayCover/en.lproj/Localizable.strings

```diff
@@ -90,6 +90,8 @@
 "playapp.install.signing" = "Signing app...";
 "playapp.install.addToLib" = "Adding app to library...";
 "playapp.install.copy" = "Copying app...";
+"playapp.uninstall.begin" = "Uninstalling app...";
+"playapp.uninstall.finished" = "Finished";
```

Comment:
It would be better to use `playapp.progress.finished` as it is the same thing and more generic for progresses.